lang/Makefile

```diff
@@ -149,7 +151,8 @@ STAT_LANGUAGES2 = \
   solidity \
   tsx \
   typescript \
-  move-on-sui
+  move-on-sui \
+  gosu
```

Comment:
The STAT_LANGUAGES1 and STAT_LANGUAGES2 are ordered and split according to alphabetic order. Can you fix this for move-on-sui, apex, and gosu? thx